Entrypoint migration fails for Redis

The controller reports this error while starting after a restore from backup:

```text
panic: migration failed to set entrypoint for system app redis-3ddbf6f5-5202-44c7-9034-4dbd823c897b [recovered]
	panic: migration failed to set entrypoint for system app redis-3ddbf6f5-5202-44c7-9034-4dbd823c897b

goroutine 1 [running]:
panic(0x80f9e0, 0xc82024f880)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/flynn/flynn/pkg/shutdown.(*handler).exit(0xc8200159e0, 0x0, 0x0, 0x0, 0x80f9e0, 0xc82024f880)
	/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/pkg/shutdown/handler.go:69 +0x13b
github.com/flynn/flynn/pkg/shutdown.Exit()
	/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/pkg/shutdown/handler.go:40 +0x64
panic(0x80f9e0, 0xc82024f880)
	/usr/local/go/src/runtime/panic.go:443 +0x4e9
main.migrateProcessArgs(0xc8201100a8, 0x0, 0x0)
	/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/controller/schema.go:457 +0x165d
github.com/flynn/flynn/pkg/postgres.Migrations.Migrate(0xc8200dc400, 0x13, 0x20, 0xc8200e6bf0, 0x0, 0x0)
	/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/pkg/postgres/migrate.go:65 +0x4ad
main.migrateDB(0xc8200e6bf0, 0x0, 0x0)
	/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/controller/schema.go:395 +0x4e
main.main()
/tmp/tmp.2NQ96JmNFN/src/github.com/flynn/flynn/controller/controller.go:63 +0x33e
```

/cc @lmars

--- a/controller/schema.go
+++ b/controller/schema.go
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/flynn/flynn/pkg/postgres"
 )
@@ -454,7 +455,11 @@ func migrateProcessArgs(tx *postgres.DBTx) error {
 				case "logaggregator":
 					proc["entrypoint"] = []interface{}{"/bin/logaggregator"}
 				default:
-					panic(fmt.Sprintf("migration failed to set entrypoint for system app %s", *release.AppName))
+					if strings.HasPrefix(*release.AppName, "redis-") {
+						proc["entrypoint"] = []interface{}{"/bin/start-flynn-redis"}
+					} else {
+						panic(fmt.Sprintf("migration failed to set entrypoint for system app %s", *release.AppName))
+					}
 				}
 			}
 

